postownik/app/client/src/PostsList.svelte

```diff
@@ -0,0 +1,144 @@
+      {title: "tytuł", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean convallis nisl id erat mattis maximus. Phasellus porta nibh sed dapibus finibus. Nam purus urna, hendrerit vel maximus ac, ultricies nec purus. Maecenas tempus lectus dui, ut pretium nunc viverra eget. Vivamus mollis dolor in enim dignissim imperdiet. Maecenas dui arcu, pulvinar vitae nunc eget, dignissim aliquam lacus. Quisque consectetur sollicitudin lacus ac euismod. Donec feugiat orci at elit semper, accumsan maximus massa consectetur. Donec ultrices risus justo, eget consectetur odio dapibus at. Curabitur purus odio, posuere quis ligula a, tincidunt faucibus ante. Sed dignissim est sit amet vehicula sollicitudin. Nullam sollicitudin quis neque in dictum.", type: "private", visible: true},
+      {title: "tytuł", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean convallis nisl id erat mattis maximus. Phasellus porta nibh sed dapibus finibus. Nam purus urna, hendrerit vel maximus ac, ultricies nec purus. Maecenas tempus lectus dui, ut pretium nunc viverra eget. Vivamus mollis dolor in enim dignissim imperdiet. Maecenas dui arcu, pulvinar vitae nunc eget, dignissim aliquam lacus. Quisque consectetur sollicitudin lacus ac euismod. Donec feugiat orci at elit semper, accumsan maximus massa consectetur. Donec ultrices risus justo, eget consectetur odio dapibus at. Curabitur purus odio, posuere quis ligula a, tincidunt faucibus ante. Sed dignissim est sit amet vehicula sollicitudin. Nullam sollicitudin quis neque in dictum.", type: "private", visible: true},
+      {title: "tytuł", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean convallis nisl id erat mattis maximus. Phasellus porta nibh sed dapibus finibus. Nam purus urna, hendrerit vel maximus ac, ultricies nec purus. Maecenas tempus lectus dui, ut pretium nunc viverra eget. Vivamus mollis dolor in enim dignissim imperdiet. Maecenas dui arcu, pulvinar vitae nunc eget, dignissim aliquam lacus. Quisque consectetur sollicitudin lacus ac euismod. Donec feugiat orci at elit semper, accumsan maximus massa consectetur. Donec ultrices risus justo, eget consectetur odio dapibus at. Curabitur purus odio, posuere quis ligula a, tincidunt faucibus ante. Sed dignissim est sit amet vehicula sollicitudin. Nullam sollicitudin quis neque in dictum.", type: "concept", visible: true}
+
+      ];
+      let selectedType;
+        $:{if(selectedType == 0){
+          posts.forEach((post) =>{post.visible = true})
+        }
+        else if(selectedType == 1){
+          posts.forEach((post) =>{
+            if(post.type != "private")
+            post.visible = false
+            else
+            post.visible = true
```

Comment:
takie coś można zapisać nieco sprawniej
`post.visible = post.type == 'private'` xD

Reply:
ale też tak jak pisałem wyżej - to czy post będzie w liście postów będzie filtrowane już na backendzie przez filtry które wyślecie (przez filtr `status`)